### ISSUE TITLE
MACSYMA : describe command not working

### ISSUE THREAD

**Issue by TheFausap** [PDP-10/its#958]
Hello,

I'm trying to use describe command (in MACSYMA), but it seems a file is missing:

```
(C14) describe("integ");
[MACSYM, BINDEX, DSK, MAXOUT]  FILE NOT FOUND
```

the file is in its-vault at files/manual/macsym.bindex

Is it possible to include this in the build ?

thanks,
Fausto

**Comment by larsbrinkhoff** [comment]
Yes, it is.  Make the directory doc/maxout and copy macsym.bindex there.  Add `maxout` to the DOC variable in the Makefile.
